x/ccv/parent/keeper/proposal.go

```diff
@@ -6,10 +6,15 @@ import (
 	"time"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
 	clienttypes "github.com/cosmos/ibc-go/modules/core/02-client/types"
 	commitmenttypes "github.com/cosmos/ibc-go/modules/core/23-commitment/types"
 	ibctmtypes "github.com/cosmos/ibc-go/modules/light-clients/07-tendermint/types"
 	"github.com/cosmos/interchain-security/x/ccv/parent/types"
+	tmtypes "github.com/tendermint/tendermint/abci/types"
```

Comment:
For consistency with the rest of the implementation, use `abci` instead of `tmtypes` (see parent/relayer.go). It's also `abci` throughout cosmos-sdk https://github.com/cosmos/cosmos-sdk/blob/056db1295768cbf4675b8025b66d83bbaadd923b/x/staking/genesis.go#L6